### PR TITLE
Issue#59

### PR DESCRIPTION
**Pull request by sachaboudjema**
Hi, updated the arubaclearpass_config code to support API authentication dirctly with a token

### CODE REVIEW

**Comment by alagoutte** [comment]
Merci for fix also the wrong example for CPPM ;-)

**Comment by karthikeyan-dhandapani** [comment]
Fixes #59

File: aruba_module_installer/library/modules/network/aruba_clearpass/arubaclearpass_config.py [hunk 1]
```diff
@@ -154,9 +167,30 @@ def main():
     method = module.params.get('method')
     data = module.params.get('data')
     module.cookie_file = "aruba_cookie.pkl"
```
Comment (karthikeyan-dhandapani) [inline aruba_module_installer/library/modules/network/aruba_clearpass/arubaclearpass_config.py]:
```suggestion  
    module.cookie_file = "aruba_cookie.pkl"  
    access_token = module.params.get('access_token')
    module.api_call = {}
```

Comment (karthikeyan-dhandapani) [reply aruba_module_installer/library/modules/network/aruba_clearpass/arubaclearpass_config.py]: Need to define the variables access_token and module.api_call before using them

**Comment by karthikeyan-dhandapani** [comment]
Would also like to update the examples in the [module documentation](https://github.com/aruba/aruba-ansible-modules/blob/master/module_documentation/aruba_clearpass/arubaclearpass_config.md
) folder with this PR!

**Comment by sachaboudjema** [comment]
Corrected missing variable assignment and updated documentation

**Comment by karthikeyan-dhandapani** [comment]
Thanks for the commit

Still receiving this error, 
aruba_clearpass/arubaclearpass_config.py", line 81, in login_cppm
AttributeError: 'AnsibleModule' object has no attribute 'api_call'

Could you please add, ```module.api_call = {}``` in addition to access_token?

**Comment by sachaboudjema** [comment]
Fixed

**Pull request merged**